packages/actions/resources/views/components/group.blade.php

```diff
@@ -7,6 +7,7 @@
     'dropdownMaxHeight' => null,
     'dropdownPlacement' => null,
     'dropdownWidth' => null,
+    'dropdownOffset' => 8,
```

Comment:
Please remove the default here, so we have it just in a single place (the actual dropdown Blade component).